### ISSUE TITLE
consider refactor xmas tree svg to html

### ISSUE THREAD

**Issue by lauraGgit** [18F/fs-permit-platform#142]
## Notes
originally used svg to have better print rendering but probably could refactor more simply to just html and then convert to pdf/png

## Acceptance Criteria
- [ ]

## Definition of Done
- [ ] Code unit / e2e tested and comply with QWASP
- [ ] No new security vulnerabilities
- [ ] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated
- [ ] Docker updated
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by lauraGgit** [comment]
## What it would take
- swap out [Permit Design SVG](https://github.com/18F/fs-open-forest-platform/blob/dev/server/src/templates/christmas-trees/permit-design.svg) with HTML and CSS
- update [applicant info](https://github.com/18F/fs-open-forest-platform/blob/dev/server/src/services/christmas-trees-permit-svg-util.es6#L27) and [appForestSpecificInfo](https://github.com/18F/fs-open-forest-platform/blob/dev/server/src/services/christmas-trees-permit-svg-util.es6#L52)
- switch out from svg2png to [dom-to-image](https://www.npmjs.com/package/dom-to-image) in the [generatePng](https://github.com/18F/fs-open-forest-platform/blob/dev/server/src/services/christmas-trees-permit-svg-util.es6#L121)
- separate out the rules functions into own service


## What we would get
- have more consistent with the rules generation
- easier accessibility annotations?
- more updatable?
- greatly simplify reading in of [template](https://github.com/18F/fs-open-forest-platform/blob/dev/server/src/services/christmas-trees-permit-svg-util.es6#L96)

## TLDR
Like worth waiting to do see if we'd want to do this until after the xmas season when we would potentially need to change the permit anyway.

**Comment by msadakFS** [comment]
@lauraGgit Thanks for the breakdown above. Looking like the decision here is to put this into the icebox.
